ecs_deploy/ecs.py

```diff
@@ -503,6 +886,92 @@ def set_execution_role_arn(self, execution_role_arn):
             self.execution_role_arn = execution_role_arn
             self._diff.append(diff)
 
+    def set_volumes(self, volumes_list):
+        volumes = []
+
+        for volume in volumes_list:
+            mapping = {}
+            mapping["name"] = volume[0]
+            mapping.setdefault("host",{})
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

You are using the `dict.setdefault()` method. We recommend using the `defaultdict` object to handle missing items in internal state because the default value is automatically generated when you access or modify a missing key. `defaultdict` can also be easier to use and understand.

[Learn more](https://docs.python.org/3/library/collections.html#collections.defaultdict)

Reply:
Should be fine now.